src/js/hexbin/HexbinLayer.js

```diff
@@ -302,10 +302,30 @@ L.HexbinLayer = (L.Layer ? L.Layer : L.Class).extend({
 		this.options.onclick = clickFn;
 		//this._redraw();
 		return this;
+	},
+
+	/*
+	 * Returns an array of the points in the path, or nested arrays of points in case of multi-polyline.
+	 */
+	getLatLngs: function () {
+		return this._data.map(function(d) { return L.latLng(d[0], d[1]); });
+	},
+
+	/*
+	 * Get path geometry as GeoJSON
+	 */
+	toGeoJSON: function () {
+		var latLngs = this._data.map(function(d) { return L.latLng(d[0], d[1]); });
```

Comment:
I'll still merge it. I'll just remove the conditional that checks to see if it's a multi-line. I could refactor it in the future to store/maintain multiple LineStrings